### PR TITLE
Remove hostPort from charts

### PR DESCRIPTION
**Pull request by daniel-lau-mf**
These have never been required and are leftover from
very early development work

Signed-off-by: daniel-lau-mf <36928012+daniel-lau-mf@users.noreply.github.com>

### CODE REVIEW

File: helm/basic-idol/charts/basic-idol/templates/deployment.yaml [hunk 1]
```diff
@@ -61,10 +61,8 @@ spec:
 {{- template "basicidol.deployment.standardLivenessProbe" }}
         ports:
         - containerPort: {{ (index .Values.content.ports 0).container | int }}
-          hostPort: {{ (index .Values.content.ports 0).host | int }}
```
Comment (christopher-rayson-mf) [inline helm/basic-idol/charts/basic-idol/templates/deployment.yaml:64]: Does this mean we can also remove `content.ports.host` etc. from the Values YAML file (and similarly for distributed-idol)?

Comment (daniel-lau-mf) [reply helm/basic-idol/charts/basic-idol/templates/deployment.yaml:64]: [1336c3a](https://github.com/opentext-idol/idol-containers-toolkit/pull/9/commits/1336c3afe30259e4dd69e39f91d996c299c07939)

Comment (christopher-rayson-mf) [reply helm/basic-idol/charts/basic-idol/templates/deployment.yaml:64]: Was a bit confused about the definitions of `custommetricsPorts` versus `prometheusPorts` in distributed-idol/values.yaml — the former has `.service` for the service's port and `.port` for the container port; the latter uses `.port` for the service and `.container` for the target. Think they'd be more consistent (with each other and everything else) if they both used `.service` and `.container`.

(Possibly our service definitions should reference the pod ports [by name](https://kubernetes.io/docs/concepts/services-networking/service/#field-spec-ports), and similarly for the [Ingress rules](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-rules), but that's beyond the scope of these changes. Using helm keeps us DRY here anyway...)

Comment (daniel-lau-mf) [reply helm/basic-idol/charts/basic-idol/templates/deployment.yaml:64]: [15fd04e](https://github.com/opentext-idol/idol-containers-toolkit/pull/9/commits/15fd04e2d82120f0703fd9ac6f0329f356e276fd) & [9cd95bd](https://github.com/opentext-idol/idol-containers-toolkit/pull/9/commits/9cd95bdab136ed78ebf05ca948be77b59d1e225a)

**Review by christopher-rayson-mf — APPROVED** [review]

**Pull request merged**